Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -245,23 +256,26 @@ public void centerJunctionTop(GrabberCameraPipeline pipeline) {
      * @param pipeline the RobotCameraPipeline being used by the robot camera
      */
     public void centerConeStack(RobotCameraPipeline pipeline) {
-        double xOffset, width;
-
+        double xOffset, width, strafePower, turnPower;
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
+            xOffset = Constants.CAMERA_CENTER_X-pipeline.xPosition;
```

Comment:
Why is this backwards from the other `xOffset`?
This is also easier to read with the spaces
```suggestion
            xOffset = Constants.CAMERA_CENTER_X - pipeline.xPosition;
```

Reply:
Clarified and edited.

Reply:
no longer clarified or edited